[2021-12-20 11:19:19,026] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_louth.scrap_data scheduled__2021-12-19T10:15:00+00:00 [queued]>
[2021-12-20 11:19:25,281] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_louth.scrap_data scheduled__2021-12-19T10:15:00+00:00 [queued]>
[2021-12-20 11:19:25,560] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-20 11:19:25,672] {taskinstance.py:1242} INFO - Starting attempt 4 of 4
[2021-12-20 11:19:25,828] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-20 11:19:32,094] {taskinstance.py:1262} INFO - Executing <Task(SparkSubmitOperator): scrap_data> on 2021-12-19 10:15:00+00:00
[2021-12-20 11:19:34,291] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'scraping_daft_louth', 'scrap_data', 'scheduled__2021-12-19T10:15:00+00:00', '--job-id', '3434', '--raw', '--subdir', 'DAGS_FOLDER/scraping_daft_louth.py', '--cfg-path', '/tmp/tmp0z_4ztut', '--error-file', '/tmp/tmpg19mlwuc']
[2021-12-20 11:19:34,390] {standard_task_runner.py:77} INFO - Job 3434: Subtask scrap_data
[2021-12-20 11:19:33,494] {standard_task_runner.py:52} INFO - Started process 5637 to run task
[2021-12-20 11:19:47,951] {logging_mixin.py:109} INFO - Running <TaskInstance: scraping_daft_louth.scrap_data scheduled__2021-12-19T10:15:00+00:00 [running]> on host a35fe600e898
[2021-12-20 11:20:01,433] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clement.liscoet@gmail.com
AIRFLOW_CTX_DAG_OWNER=clemoni
AIRFLOW_CTX_DAG_ID=scraping_daft_louth
AIRFLOW_CTX_TASK_ID=scrap_data
AIRFLOW_CTX_EXECUTION_DATE=2021-12-19T10:15:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-19T10:15:00+00:00
[2021-12-20 11:20:03,481] {base.py:79} INFO - Using connection to: id: spark_default. Host: spark://spark:7077, Port: None, Schema: , Login: ***, Password: ***, extra: {}
[2021-12-20 11:20:04,297] {spark_submit.py:360} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py louth
[2021-12-20 11:23:06,990] {spark_submit.py:514} INFO - Using properties file: null
[2021-12-20 11:23:10,669] {spark_submit.py:514} INFO - WARNING: An illegal reflective access operation has occurred
[2021-12-20 11:23:10,672] {spark_submit.py:514} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.6/site-packages/pyspark/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2021-12-20 11:23:10,681] {spark_submit.py:514} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2021-12-20 11:23:10,687] {spark_submit.py:514} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2021-12-20 11:23:10,728] {spark_submit.py:514} INFO - WARNING: All illegal access operations will be denied in a future release
[2021-12-20 11:23:13,435] {spark_submit.py:514} INFO - Parsed arguments:
[2021-12-20 11:23:13,440] {spark_submit.py:514} INFO - master                  spark://spark:7077
[2021-12-20 11:23:13,443] {spark_submit.py:514} INFO - deployMode              null
[2021-12-20 11:23:13,445] {spark_submit.py:514} INFO - executorMemory          null
[2021-12-20 11:23:13,448] {spark_submit.py:514} INFO - executorCores           null
[2021-12-20 11:23:13,455] {spark_submit.py:514} INFO - totalExecutorCores      null
[2021-12-20 11:23:13,459] {spark_submit.py:514} INFO - propertiesFile          null
[2021-12-20 11:23:13,460] {spark_submit.py:514} INFO - driverMemory            null
[2021-12-20 11:23:13,465] {spark_submit.py:514} INFO - driverCores             null
[2021-12-20 11:23:13,467] {spark_submit.py:514} INFO - driverExtraClassPath    null
[2021-12-20 11:23:13,469] {spark_submit.py:514} INFO - driverExtraLibraryPath  null
[2021-12-20 11:23:13,478] {spark_submit.py:514} INFO - driverExtraJavaOptions  null
[2021-12-20 11:23:13,482] {spark_submit.py:514} INFO - supervise               false
[2021-12-20 11:23:13,486] {spark_submit.py:514} INFO - queue                   null
[2021-12-20 11:23:13,488] {spark_submit.py:514} INFO - numExecutors            null
[2021-12-20 11:23:13,491] {spark_submit.py:514} INFO - files                   null
[2021-12-20 11:23:13,492] {spark_submit.py:514} INFO - pyFiles                 null
[2021-12-20 11:23:13,497] {spark_submit.py:514} INFO - archives                null
[2021-12-20 11:23:13,499] {spark_submit.py:514} INFO - mainClass               null
[2021-12-20 11:23:13,502] {spark_submit.py:514} INFO - primaryResource         file:/opt/***/jobs/get_ads_per_county.py
[2021-12-20 11:23:13,507] {spark_submit.py:514} INFO - name                    scrap data
[2021-12-20 11:23:13,513] {spark_submit.py:514} INFO - childArgs               [louth]
[2021-12-20 11:23:13,520] {spark_submit.py:514} INFO - jars                    null
[2021-12-20 11:23:13,527] {spark_submit.py:514} INFO - packages                null
[2021-12-20 11:23:13,533] {spark_submit.py:514} INFO - packagesExclusions      null
[2021-12-20 11:23:13,541] {spark_submit.py:514} INFO - repositories            null
[2021-12-20 11:23:13,552] {spark_submit.py:514} INFO - verbose                 true
[2021-12-20 11:23:13,561] {spark_submit.py:514} INFO - 
[2021-12-20 11:23:13,599] {spark_submit.py:514} INFO - Spark properties used, including those specified through
[2021-12-20 11:23:13,624] {spark_submit.py:514} INFO - --conf and those from the properties file null:
[2021-12-20 11:23:13,626] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-20 11:23:13,631] {spark_submit.py:514} INFO - 
[2021-12-20 11:23:13,639] {spark_submit.py:514} INFO - 
[2021-12-20 11:23:31,954] {spark_submit.py:514} INFO - Main class:
[2021-12-20 11:23:32,017] {spark_submit.py:514} INFO - org.apache.spark.deploy.PythonRunner
[2021-12-20 11:23:32,073] {spark_submit.py:514} INFO - Arguments:
[2021-12-20 11:23:32,091] {spark_submit.py:514} INFO - file:/opt/***/jobs/get_ads_per_county.py
[2021-12-20 11:23:32,092] {spark_submit.py:514} INFO - null
[2021-12-20 11:23:32,093] {spark_submit.py:514} INFO - louth
[2021-12-20 11:23:32,108] {spark_submit.py:514} INFO - --verbose
[2021-12-20 11:23:32,116] {spark_submit.py:514} INFO - Spark config:
[2021-12-20 11:23:32,155] {spark_submit.py:514} INFO - (spark.app.name,scrap data)
[2021-12-20 11:23:32,170] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-20 11:23:32,181] {spark_submit.py:514} INFO - (spark.submit.pyFiles,)
[2021-12-20 11:23:32,195] {spark_submit.py:514} INFO - (spark.submit.deployMode,client)
[2021-12-20 11:23:32,196] {spark_submit.py:514} INFO - Classpath elements:
[2021-12-20 11:23:32,201] {spark_submit.py:514} INFO - 
[2021-12-20 11:23:32,222] {spark_submit.py:514} INFO - 
[2021-12-20 11:23:32,228] {spark_submit.py:514} INFO - 
[2021-12-20 11:23:46,343] {spark_submit.py:514} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2021-12-20 11:23:46,415] {spark_submit.py:514} INFO - 21/12/20 11:23:46 INFO SparkContext: Running Spark version 3.2.0
[2021-12-20 11:23:47,586] {spark_submit.py:514} INFO - 21/12/20 11:23:47 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2021-12-20 11:23:48,772] {spark_submit.py:514} INFO - 21/12/20 11:23:48 INFO ResourceUtils: ==============================================================
[2021-12-20 11:23:48,794] {spark_submit.py:514} INFO - 21/12/20 11:23:48 INFO ResourceUtils: No custom resources configured for spark.driver.
[2021-12-20 11:23:48,803] {spark_submit.py:514} INFO - 21/12/20 11:23:48 INFO ResourceUtils: ==============================================================
[2021-12-20 11:23:48,810] {spark_submit.py:514} INFO - 21/12/20 11:23:48 INFO SparkContext: Submitted application: Get Ads per county
[2021-12-20 11:23:49,404] {spark_submit.py:514} INFO - 21/12/20 11:23:49 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2021-12-20 11:23:49,653] {spark_submit.py:514} INFO - 21/12/20 11:23:49 INFO ResourceProfile: Limiting resource is cpu
[2021-12-20 11:23:49,658] {spark_submit.py:514} INFO - 21/12/20 11:23:49 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2021-12-20 11:23:50,487] {spark_submit.py:514} INFO - 21/12/20 11:23:50 INFO SecurityManager: Changing view acls to: ***
[2021-12-20 11:23:50,494] {spark_submit.py:514} INFO - 21/12/20 11:23:50 INFO SecurityManager: Changing modify acls to: ***
[2021-12-20 11:23:50,499] {spark_submit.py:514} INFO - 21/12/20 11:23:50 INFO SecurityManager: Changing view acls groups to:
[2021-12-20 11:23:50,501] {spark_submit.py:514} INFO - 21/12/20 11:23:50 INFO SecurityManager: Changing modify acls groups to:
[2021-12-20 11:23:50,645] {spark_submit.py:514} INFO - 21/12/20 11:23:50 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(***); groups with view permissions: Set(); users  with modify permissions: Set(***); groups with modify permissions: Set()
[2021-12-20 11:23:53,975] {spark_submit.py:514} INFO - 21/12/20 11:23:53 INFO Utils: Successfully started service 'sparkDriver' on port 34675.
[2021-12-20 11:23:54,472] {spark_submit.py:514} INFO - 21/12/20 11:23:54 INFO SparkEnv: Registering MapOutputTracker
[2021-12-20 11:23:55,281] {spark_submit.py:514} INFO - 21/12/20 11:23:55 INFO SparkEnv: Registering BlockManagerMaster
[2021-12-20 11:23:55,520] {spark_submit.py:514} INFO - 21/12/20 11:23:55 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2021-12-20 11:23:55,535] {spark_submit.py:514} INFO - 21/12/20 11:23:55 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2021-12-20 11:23:55,580] {spark_submit.py:514} INFO - 21/12/20 11:23:55 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2021-12-20 11:23:55,903] {spark_submit.py:514} INFO - 21/12/20 11:23:55 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-64e8b226-7c79-479f-958b-556fc7b58b3b
[2021-12-20 11:23:56,173] {spark_submit.py:514} INFO - 21/12/20 11:23:56 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2021-12-20 11:23:56,300] {spark_submit.py:514} INFO - 21/12/20 11:23:56 INFO SparkEnv: Registering OutputCommitCoordinator
[2021-12-20 11:23:58,458] {spark_submit.py:514} INFO - 21/12/20 11:23:58 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2021-12-20 11:23:58,473] {spark_submit.py:514} INFO - 21/12/20 11:23:58 WARN Utils: Service 'SparkUI' could not bind on port 4041. Attempting port 4042.
[2021-12-20 11:23:58,488] {spark_submit.py:514} INFO - 21/12/20 11:23:58 WARN Utils: Service 'SparkUI' could not bind on port 4042. Attempting port 4043.
[2021-12-20 11:23:58,510] {spark_submit.py:514} INFO - 21/12/20 11:23:58 WARN Utils: Service 'SparkUI' could not bind on port 4043. Attempting port 4044.
[2021-12-20 11:23:58,696] {spark_submit.py:514} INFO - 21/12/20 11:23:58 INFO Utils: Successfully started service 'SparkUI' on port 4044.
[2021-12-20 11:23:59,658] {spark_submit.py:514} INFO - 21/12/20 11:23:59 INFO SparkUI: Bound SparkUI to 0.0.0.0, and started at http://a35fe600e898:4044
[2021-12-20 11:24:02,171] {spark_submit.py:514} INFO - 21/12/20 11:24:02 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-20 11:24:02,668] {spark_submit.py:514} INFO - 21/12/20 11:24:02 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-20 11:24:02,669] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-20 11:24:02,670] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-20 11:24:02,672] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-20 11:24:02,674] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-20 11:24:02,675] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-20 11:24:02,676] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-20 11:24:02,677] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-20 11:24:02,678] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-20 11:24:02,679] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-20 11:24:02,680] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-20 11:24:02,681] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-20 11:24:02,681] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-20 11:24:02,682] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-20 11:24:02,684] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-20 11:24:02,685] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-20 11:24:02,685] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-20 11:24:02,686] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-20 11:24:02,687] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-20 11:24:02,688] {spark_submit.py:514} INFO - ... 4 more
[2021-12-20 11:24:02,689] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-20 11:24:02,690] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-20 11:24:02,691] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-20 11:24:02,692] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-20 11:24:02,693] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-20 11:24:02,694] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-20 11:24:02,695] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-20 11:24:02,696] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-20 11:24:02,697] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-20 11:24:02,697] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-20 11:24:02,698] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-20 11:24:02,699] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-20 11:24:02,700] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-20 11:24:02,701] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-20 11:24:02,702] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-20 11:24:02,703] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-20 11:24:02,704] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-20 11:24:02,705] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-20 11:24:02,706] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-20 11:24:02,707] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-20 11:24:02,708] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-20 11:24:02,709] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-20 11:24:02,710] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-20 11:24:02,711] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-20 11:24:02,712] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-20 11:24:02,713] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-20 11:24:02,714] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-20 11:24:02,715] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-20 11:24:02,716] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-20 11:24:02,717] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-20 11:24:02,718] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-20 11:24:02,719] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-20 11:24:02,721] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-20 11:24:02,722] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-20 11:24:02,723] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-20 11:24:02,724] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-20 11:24:02,725] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-20 11:24:02,727] {spark_submit.py:514} INFO - ... 1 more
[2021-12-20 11:24:22,016] {spark_submit.py:514} INFO - 21/12/20 11:24:22 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-20 11:24:22,050] {spark_submit.py:514} INFO - 21/12/20 11:24:22 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-20 11:24:22,051] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-20 11:24:22,052] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-20 11:24:22,053] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-20 11:24:22,054] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-20 11:24:22,055] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-20 11:24:22,057] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-20 11:24:22,058] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-20 11:24:22,059] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-20 11:24:22,060] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-20 11:24:22,061] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-20 11:24:22,062] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-20 11:24:22,063] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-20 11:24:22,064] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-20 11:24:22,065] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-20 11:24:22,066] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-20 11:24:22,067] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-20 11:24:22,068] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-20 11:24:22,070] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-20 11:24:22,071] {spark_submit.py:514} INFO - ... 4 more
[2021-12-20 11:24:22,072] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-20 11:24:22,073] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-20 11:24:22,075] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-20 11:24:22,076] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-20 11:24:22,079] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-20 11:24:22,080] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-20 11:24:22,083] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-20 11:24:22,084] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-20 11:24:22,085] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-20 11:24:22,087] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-20 11:24:22,088] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-20 11:24:22,089] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-20 11:24:22,090] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-20 11:24:22,092] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-20 11:24:22,093] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-20 11:24:22,095] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-20 11:24:22,096] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-20 11:24:22,100] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-20 11:24:22,101] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-20 11:24:22,110] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-20 11:24:22,114] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-20 11:24:22,115] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-20 11:24:22,117] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-20 11:24:22,118] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-20 11:24:22,119] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-20 11:24:22,121] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-20 11:24:22,122] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-20 11:24:22,122] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-20 11:24:22,124] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-20 11:24:22,125] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-20 11:24:22,126] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-20 11:24:22,127] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-20 11:24:22,128] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-20 11:24:22,129] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-20 11:24:22,130] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-20 11:24:22,132] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-20 11:24:22,133] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-20 11:24:22,135] {spark_submit.py:514} INFO - ... 1 more
[2021-12-20 11:24:41,981] {spark_submit.py:514} INFO - 21/12/20 11:24:41 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-20 11:24:42,025] {spark_submit.py:514} INFO - 21/12/20 11:24:42 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-20 11:24:42,027] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-20 11:24:42,037] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-20 11:24:42,048] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-20 11:24:42,053] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-20 11:24:42,055] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-20 11:24:42,057] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-20 11:24:42,059] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-20 11:24:42,062] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-20 11:24:42,066] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-20 11:24:42,068] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-20 11:24:42,072] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-20 11:24:42,073] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-20 11:24:42,075] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-20 11:24:42,077] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-20 11:24:42,078] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-20 11:24:42,079] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-20 11:24:42,080] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-20 11:24:42,081] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-20 11:24:42,082] {spark_submit.py:514} INFO - ... 4 more
[2021-12-20 11:24:42,083] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-20 11:24:42,084] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-20 11:24:42,085] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-20 11:24:42,087] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-20 11:24:42,088] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-20 11:24:42,089] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-20 11:24:42,090] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-20 11:24:42,091] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-20 11:24:42,092] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-20 11:24:42,093] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-20 11:24:42,096] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-20 11:24:42,099] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-20 11:24:42,101] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-20 11:24:42,106] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-20 11:24:42,109] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-20 11:24:42,110] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-20 11:24:42,112] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-20 11:24:42,117] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-20 11:24:42,119] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-20 11:24:42,121] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-20 11:24:42,124] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-20 11:24:42,125] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-20 11:24:42,126] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-20 11:24:42,129] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-20 11:24:42,133] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-20 11:24:42,135] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-20 11:24:42,137] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-20 11:24:42,140] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-20 11:24:42,142] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-20 11:24:42,145] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-20 11:24:42,146] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-20 11:24:42,148] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-20 11:24:42,150] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-20 11:24:42,153] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-20 11:24:42,155] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-20 11:24:42,156] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-20 11:24:42,158] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-20 11:24:42,160] {spark_submit.py:514} INFO - ... 1 more
[2021-12-20 11:25:01,991] {spark_submit.py:514} INFO - 21/12/20 11:25:01 ERROR StandaloneSchedulerBackend: Application has been killed. Reason: All masters are unresponsive! Giving up.
[2021-12-20 11:25:01,999] {spark_submit.py:514} INFO - 21/12/20 11:25:01 WARN StandaloneSchedulerBackend: Application ID is not initialized yet.
[2021-12-20 11:25:02,064] {spark_submit.py:514} INFO - 21/12/20 11:25:02 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 33825.
[2021-12-20 11:25:02,066] {spark_submit.py:514} INFO - 21/12/20 11:25:02 INFO NettyBlockTransferService: Server created on a35fe600e898:33825
[2021-12-20 11:25:02,079] {spark_submit.py:514} INFO - 21/12/20 11:25:02 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2021-12-20 11:25:02,089] {spark_submit.py:514} INFO - 21/12/20 11:25:02 INFO SparkUI: Stopped Spark web UI at http://a35fe600e898:4044
[2021-12-20 11:25:02,117] {spark_submit.py:514} INFO - 21/12/20 11:25:02 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, a35fe600e898, 33825, None)
[2021-12-20 11:25:02,131] {spark_submit.py:514} INFO - 21/12/20 11:25:02 INFO StandaloneSchedulerBackend: Shutting down all executors
[2021-12-20 11:25:02,151] {spark_submit.py:514} INFO - 21/12/20 11:25:02 INFO BlockManagerMasterEndpoint: Registering block manager a35fe600e898:33825 with 434.4 MiB RAM, BlockManagerId(driver, a35fe600e898, 33825, None)
[2021-12-20 11:25:02,163] {spark_submit.py:514} INFO - 21/12/20 11:25:02 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Asking each executor to shut down
[2021-12-20 11:25:02,168] {spark_submit.py:514} INFO - 21/12/20 11:25:02 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, a35fe600e898, 33825, None)
[2021-12-20 11:25:02,189] {spark_submit.py:514} INFO - 21/12/20 11:25:02 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, a35fe600e898, 33825, None)
[2021-12-20 11:25:02,209] {spark_submit.py:514} INFO - 21/12/20 11:25:02 WARN StandaloneAppClient$ClientEndpoint: Drop UnregisterApplication(null) because has not yet connected to master
[2021-12-20 11:25:02,342] {spark_submit.py:514} INFO - 21/12/20 11:25:02 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2021-12-20 11:25:02,433] {spark_submit.py:514} INFO - 21/12/20 11:25:02 INFO MemoryStore: MemoryStore cleared
[2021-12-20 11:25:02,438] {spark_submit.py:514} INFO - 21/12/20 11:25:02 INFO BlockManager: BlockManager stopped
[2021-12-20 11:25:02,466] {spark_submit.py:514} INFO - 21/12/20 11:25:02 INFO BlockManagerMaster: BlockManagerMaster stopped
[2021-12-20 11:25:02,473] {spark_submit.py:514} INFO - 21/12/20 11:25:02 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2021-12-20 11:25:02,536] {spark_submit.py:514} INFO - 21/12/20 11:25:02 INFO SparkContext: Successfully stopped SparkContext
[2021-12-20 11:25:02,983] {spark_submit.py:514} INFO - 21/12/20 11:25:02 ERROR SparkContext: Error initializing SparkContext.
[2021-12-20 11:25:02,986] {spark_submit.py:514} INFO - java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-20 11:25:02,988] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-20 11:25:02,990] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-20 11:25:02,991] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-20 11:25:02,992] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-20 11:25:02,994] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-20 11:25:02,996] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-20 11:25:02,997] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-20 11:25:02,999] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-20 11:25:03,001] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-20 11:25:03,006] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-20 11:25:03,012] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-20 11:25:03,013] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-20 11:25:03,019] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-20 11:25:03,023] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-20 11:25:03,026] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-20 11:25:03,027] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-20 11:25:03,029] {spark_submit.py:514} INFO - 21/12/20 11:25:02 INFO SparkContext: SparkContext already stopped.
[2021-12-20 11:25:03,032] {spark_submit.py:514} INFO - Traceback (most recent call last):
[2021-12-20 11:25:03,035] {spark_submit.py:514} INFO - File "/opt/***/jobs/get_ads_per_county.py", line 13, in <module>
[2021-12-20 11:25:03,038] {spark_submit.py:514} INFO - .appName("Get Ads per county") \
[2021-12-20 11:25:03,042] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/session.py", line 228, in getOrCreate
[2021-12-20 11:25:03,044] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 392, in getOrCreate
[2021-12-20 11:25:03,046] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 147, in __init__
[2021-12-20 11:25:03,048] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 209, in _do_init
[2021-12-20 11:25:03,051] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 329, in _initialize_context
[2021-12-20 11:25:03,054] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/java_gateway.py", line 1574, in __call__
[2021-12-20 11:25:03,059] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/protocol.py", line 328, in get_return_value
[2021-12-20 11:25:03,144] {spark_submit.py:514} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling None.org.apache.spark.api.java.JavaSparkContext.
[2021-12-20 11:25:03,157] {spark_submit.py:514} INFO - : java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-20 11:25:03,166] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-20 11:25:03,175] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-20 11:25:03,184] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-20 11:25:03,193] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-20 11:25:03,201] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-20 11:25:03,204] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-20 11:25:03,207] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-20 11:25:03,210] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-20 11:25:03,220] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-20 11:25:03,221] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-20 11:25:03,225] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-20 11:25:03,227] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-20 11:25:03,230] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-20 11:25:03,234] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-20 11:25:03,263] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-20 11:25:03,266] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-20 11:25:03,269] {spark_submit.py:514} INFO - 
[2021-12-20 11:25:03,961] {spark_submit.py:514} INFO - 21/12/20 11:25:03 INFO ShutdownHookManager: Shutdown hook called
[2021-12-20 11:25:03,973] {spark_submit.py:514} INFO - 21/12/20 11:25:03 INFO ShutdownHookManager: Deleting directory /tmp/spark-ea38d856-205a-4547-87d0-4d120d9068ec
[2021-12-20 11:25:04,028] {spark_submit.py:514} INFO - 21/12/20 11:25:03 INFO ShutdownHookManager: Deleting directory /tmp/spark-37a341cd-f458-4b48-89e4-0d5bd1e231f1
[2021-12-20 11:25:05,924] {taskinstance.py:1703} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py louth. Error code is: 1.
[2021-12-20 11:25:05,976] {taskinstance.py:1280} INFO - Marking task as FAILED. dag_id=scraping_daft_louth, task_id=scrap_data, execution_date=20211219T101500, start_date=20211220T111919, end_date=20211220T112505
[2021-12-20 11:25:06,161] {warnings.py:99} WARNING - /home/***/.local/lib/python3.6/site-packages/***/utils/email.py:102: PendingDeprecationWarning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
  send_mime_email(e_from=smtp_mail_from, e_to=recipients, mime_msg=msg, conn_id=conn_id, dryrun=dryrun)

[2021-12-20 11:25:06,163] {email.py:208} INFO - Email alerting: attempt 1
[2021-12-20 11:25:06,892] {email.py:220} INFO - Sent an alert email to ['clement.liscoet@gmail.com']
[2021-12-20 11:25:08,674] {standard_task_runner.py:91} ERROR - Failed to execute job 3434 for task scrap_data
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 292, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py louth. Error code is: 1.
[2021-12-20 11:25:08,696] {local_task_job.py:212} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2021-12-20 11:25:08,742] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 5637
[2021-12-20 11:25:08,929] {process_utils.py:66} INFO - Process psutil.Process(pid=5637, status='terminated', exitcode=1, started='11:19:33') (5637) terminated with exit code 1
